test-bench.sh

```diff
@@ -159,6 +160,9 @@ while [ $# -gt 0 ]; do
       ;;
     --jvm-args)
       JVM_ARGS_INPUT="$2"
+      ;;
+	--file-idf)
```

Comment:
Is fileID a required parameter? I think we should make it optional, and use some other identifier (epoch time) if not provided.

Reply:
No, it is an optional parameter and I have made some changes to default to an epoch timestamp. Since it will be confusing by itself in the ReadMe document, I have added a description in the helper text that can be accessed with the '--help' parameter